stable-2.4.0.md

```diff
@@ -0,0 +1,188 @@
+## stable-2.4.0
+
+This stable release introduces adds support for SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md),
```

Comment:
Come to think of it, I'm drafting the announcement blost post right now. It would be good to sync w/this first paragraph and make sure we're telling the same story.